### ISSUE TITLE
Error: Module did not self-register

### ISSUE THREAD

**Issue by lateiskate** [Automattic/node-canvas#1536]
##  Issue or Feature
I can't seem to run my app with canvas... I keep getting a **Module did not self-register** error.

```
Error: Module did not self-register: '/home/app/node_modules/canvas/build/Release/canvas.node'.
at Object.Module._extensions..node (internal/modules/cjs/loader.js:1208:18)
at Module.load (internal/modules/cjs/loader.js:1002:32)
at Function.Module._load (internal/modules/cjs/loader.js:901:14)
at Module.require (internal/modules/cjs/loader.js:1044:19)
at require (internal/modules/cjs/helpers.js:77:18)
at Object.<anonymous> (/home/app/node_modules/canvas/lib/bindings.js:3:18)
at Module._compile (internal/modules/cjs/loader.js:1158:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
at Module.load (internal/modules/cjs/loader.js:1002:32)
at Function.Module._load (internal/modules/cjs/loader.js:901:14)
at Module.require (internal/modules/cjs/loader.js:1044:19)
at require (internal/modules/cjs/helpers.js:77:18)
at Object.<anonymous> (/home/app/node_modules/canvas/lib/canvas.js:9:18)
at Module._compile (internal/modules/cjs/loader.js:1158:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
at Module.load (internal/modules/cjs/loader.js:1002:32)
```
**Something perhaps worth noting, I'm using worker_threads too, not sure if that means anything but yeah.

## Your Environment
* Canvas@2.6.1
* Node.js 12.16.1

Thank you.

**Comment by zbjornson** [comment]
Node-canvas doesn't work with worker threads yet, see #1394.

**Comment by nedasoo** [comment]
Any estimation on when worker_threads support is going to be implemented?

**Comment by joshkel** [comment]
This will be addressed by https://github.com/Automattic/node-canvas/pull/2235, which is expected to be released in node-canvas 3.0.0.
